### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by rmrmrmr**
# Portolio: add contact form

## In this milestone we:
-Added a Contact Me section
-Added text to the Contact Me section
-Added a Contact me form, which includes a name, e-mail and comment inputs
-Added functionality to the form with Formspree
-Added a box with social media links
-Added a section that includes a contact e-mail
-Added a background image to the Contact me section
-Added hover and click functionality to the buttons of the whole website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -352,46 +353,49 @@ <h3 id="abtMeText" class="abtMeText">I can help you build a product,<br>feature
...
+  <section id="contact">
     <div id="contactMeTextPH">
-      <p id="contactMeText">
+      <h2 id="contactMeText">
         I'm always interested
         in hearing about new projects,
         so if you'd like to chat please
         get in touch.
-      </p>
+      </h2>
```
Comment (thecodechaser) [inline index.html]: - [x] Please increase the font weight for the heading and make sure it matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design.

### Your design
image

### Figma design
image

File: index.html [hunk 2]
```diff
@@ -352,46 +353,49 @@ <h3 id="abtMeText" class="abtMeText">I can help you build a product,<br>feature
...
-    <form id="contactForm">
-      <input type="text" name="FullName" id="nameFormBox" class="formBox1" placeholder="Full Name">
-      <input type="text" name="EmailAddress" id="mailFormBox" class="formBox1" placeholder="Email Adress">
-      <input type="text" name="EmailAddress" id="commentsFormBox" class="formBox2" placeholder="Enter Text Here">
+    <form id="contactForm" action="https://formspree.io/f/moqbvdgn" method="post">
+      <input type="text" name="name" id="nameFormBox" class="formBox1" placeholder="Full Name" maxlength="30" required>
+      <input type="email" name="email" id="mailFormBox" class="formBox1" placeholder="Email Adress" required>
+      <textarea name="comments" id="commentsFormBox" class="formBox2"
+        maxlength="500" required>Any comments?</textarea>
       <button type="submit" id="submitFormBttn">Get in touch</button>
```
Comment (thecodechaser) [inline index.html:399]: - [x] Please decrease the width of the submit button and make sure it matches the Figma design.

### Your design
image

### Figma design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @rmrmrmr👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @rmrmrmr, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**